packages/flex-dev-utils/src/keytar.ts

```diff
@@ -0,0 +1,115 @@
+import keytar from 'keytar';
+
+import { prompt, Question } from './inquirer';
+import logger from './logger';
+import { multilineString } from './strings';
+
+const SERVICE_NAME = 'com.twilio.flex.plugins.builder';
+
+export interface AuthConfig {
+  apiKey: string;
+  apiSecret: string;
+}
+
+const apiKeyQuestion: Question = {
+  type: 'input',
+  name: 'apiKey',
+  message: 'Twilio API Key:',
+  validate: async (input: string) => {
+    return typeof input === 'string' && input.length > 0;
+  },
+};
+
+const apiSecretQuestion: Question = {
+  type: 'password',
+  name: 'apiSecret',
+  message: 'Twilio API Secret:',
+  validate: async (input: string) => {
+    return typeof input === 'string' && input.length > 0;
+  },
+};
+
+/**
+ * Warns about having more than one account in the service
+ * @param accounts
+ * @private
+ */
+const _moreThanOneAccountWarning = (accounts: string[]) => {
+  const items = logger.colors.blue(accounts.join(', '));
+  logger.warning(multilineString(
+    'You have more than one account stored in the credentials; the following accounts where found:',
+    items,
+    '',
+    'Consider running `flex-plugin clear` to reset the stored credentials.',
+    '',
+  ));
+};
+
+/**
+ * Fetches the API Key/Secret and stores them in keychain.
+ * If no credentials exists, then prompts the user to enter the credentials
+ */
+export const getCredentials = async (): Promise<AuthConfig> => {
+  const credentials = await getService();
+
+  let apiKey;
+  let apiSecret;
+
+  if (process.env.TWILIO_API_KEY) {
+    apiKey = process.env.TWILIO_API_KEY;
+  } else {
+    if (credentials.length > 1) {
```

Comment:
what about: https://www.npmjs.com/package/inquirer#list---type-list

Reply:
Ya will do that!